ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml

```diff
@@ -0,0 +1,265 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Controls.Validators 0.1
+
+StatusDropdown {
+    id: root
+    property bool ready: root.tokenAmountValue > 0 && root.tokenName !== d.defaultTokenNameText
+    property string tokenAmountValue: ""
+    property string tokenName: d.defaultTokenNameText
+    property url tokenImage: ""
+    property int operator: HoldingsDropdown.Operators.None
+    property bool withOperatorSelector: true
+
+    enum Operators {
+        None,
+        And,
+        Or
+    }
```

Comment:
I've seen this enum before somewhere in StatusQ ;)

Reply:
Yeah! Totally agree! Moved to `Utils`.